ouroboros-network/src/Ouroboros/Network/MockChain/Chain.hs

```diff
@@ -314,7 +313,39 @@ intersectChains c (bs :> b) =
        then Just p
        else intersectChains c bs
 
+-- * Conversions to/from 'ChainFrament'
+
+-- | Convert a 'ChainFragment' to a 'Chain'.
+fromChainFragment :: HasHeader block => CF.ChainFragment block -> Chain block
+fromChainFragment = fromNewestFirst . CF.toNewestFirst
 
+-- | Convert a 'Chain' to a 'ChainFragment'.
+toChainFragment :: HasHeader block => Chain block -> CF.ChainFragment block
+toChainFragment = CF.fromNewestFirst . toNewestFirst
+
+-- | Variant of 'toChainFragment' that assumes a valid chain and will not validate
+-- the construct 'ChainFragment'.
```

Comment:
`s/construct/constructed/` (that's my typo that you copied :slightly_smiling_face:)